src/components/chips/js/chipsDirective.js

```diff
@@ -67,11 +67,16 @@
    *    displayed when there is at least on item in the list
    * @param {boolean=} readonly Disables list manipulation (deleting or adding list items), hiding
    *    the input and delete buttons
-   * @param {expression} md-on-append An expression that when called expects you to return an object
-   *    representation of the chip input string.
+   * @param {expression} md-on-append An expression of form `myFunction($chip)` that when called
```

Comment:
Okay, talked briefly with Thomas, and we came up with splitting it into `md-transform-chip` and `md-on-append`.

Sound good to you? If so, I'll refactor it soon.